app/js/first/directives/title_directive.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function(app) {
+  app.directive('firstImageDirective', function() {
```

Comment:
It's a good idea to make your file name the same as your directive name.  That way when you are trying to find the code for your directive it will be easier to find.